auxx/src/Command/Help.hs

```diff
@@ -54,7 +54,7 @@ withTypeName (TypeNameEither tn1 tn2) needWrap =
     t' = withTypeName tn1 DontNeedWrap <> " | " <>
          withTypeName tn2 DontNeedWrap
 
-mkHelpMessage :: [CommandProc m] -> Text
+mkHelpMessage :: [Either Text (CommandProc m)] -> Text
```

Comment:
Since it's a WIP PR I hope you already planned to replace `Text` here with something more informative, at least include the name:

```
data UnavailableCommand = UnavailableCommand
    { ucName :: Name
    , ucReason :: Text
    }
```

Reply:
Yes, this is a planned change.